hackathon_site/event/jinja2/event/login.html

```diff
@@ -0,0 +1,39 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}Login{% endblock %}
+{% block form_button %}Login{% endblock %}
```

Comment:
A mentioned above, this is where we'd have to include the logic for submitting the form